docs_user/assemblies/assembly_prepare_director_operator_and_rhoso_for_adoption_process.adoc

```diff
@@ -0,0 +1,190 @@
+ifdef::context[:parent-context: {context}]
+
+[id="prepare_director_operator_and_rhoso_for_adoption_process"]
+
+:context: migrating-databases
+
+= Preparing director operator for adoption
+
+[NOTE]
+====
+The example use case presented here is of a single OCP cluster, being shared by the source ODPdO env. _and_ a destination RHOSO env. Two OCP master nodes would be transferred to RHOSO use, leaving OSPdO with one for the adoption process, after which the remaining one can be used by RHOSO env. as well.
```

Comment:
```suggestion
The following procedure uses a single {rhocp_long} cluster that is shared by the source OSPdO env.ironment and a destination {rhos_acro} environment. Two {OpenShiftShort} master nodes are transferred to the {rhos_acro} environment, leaving OSPdO with one node for the adoption process. After adoption, the remaining node can be used by the {rhos_acro}  environment as well.
```
I'm not sure I understand the part "leaving OSPdO with one node for the adoption process." If you've transferred two master nodes to RHOSO, doesn't that mean they've already been adopted too? 
I guess I don't understand the reason why you adopt the master nodes first and then the remaining node.

Reply:
I'll try short explanation: (appologies if somethings are unclear feel free to ask on any point)
- by "transfer nodes" I mean letting the RHOSO use the 2 master nodes for infrastructure only , and not OSPdO (which is left with one), so the two nodes are labled for RHOSO and would only be running RHOSO related resources.
- by adoption we mean injecting the OSP services' databases contents we had in ODPdO (or source env) into the new RHOSO service databases.
- so for the adoption you would need a destination (RHOSO) service with a "blank" database waiting to recieve the original database from OSPdO.
In an example, flow for a nova service migration: on old env dump the nova database into a file , stop the old nova service -> on new env start new service , inject the old database -> run some scripts on the computes nodes so that the computes will now be belonging to the new RHOSO env.

I believe the reason for letting the customer have a middle stage where the old services are still running os for step by step adoption, and letting the customers run checks and have more granular controll over the process

Reply:
Makes sense. Thank you for the explanation!